packages/financial-templates-lib/src/logger/DiscordTicketTransport.ts

```diff
@@ -89,6 +106,39 @@ export class DiscordTicketTransport extends Transport {
 
   // Use bot token for establishing a connection to Discord API.
   async login(): Promise<void> {
+    this.isFlushed = false; // Set the isFlushed to false to prevent the logger from closing while logging in.
     await this.client.login(this.botToken);
+    this.isFlushed = true;
```

Comment:
I worry that this will cause synchronization issues with the setting of the isFlushed variable is executeLogQueue. As suggested elsewhere, I think it's preferred to a) make this method private (so it can only be called inside this class and b) just use the counter method to avoid flushing anywhere inside the log call.

Reply:
fixed